lib/Runtime/Library/IntlEngineInterfaceExtensionObject.cpp

```diff
@@ -2289,7 +2289,6 @@ namespace Js
             case UDAT_DOW_LOCAL_FIELD:
                 typeString = _u("weekday"); break;
             case UDAT_AM_PM_FIELD:
-            case UDAT_AM_PM_MIDNIGHT_NOON_FIELD:
```

Comment:
With the new way of generating the format specifier, is this guaranteed to not show up?

Reply:
I think its much less likely -- I mostly took it out because I wasn't sure of a good way to resolve the compiletime-vs-runtime resolution of this field, and the assert I added after this should catch if it ever comes up. Perhaps that should be changed to a failfast or, even better in the future, a telemetry assert, so we know if/when this pops up in the future?